src/Symfony/Component/ErrorHandler/ThrowableUtils.php

```diff
@@ -11,14 +11,19 @@
 
 namespace Symfony\Component\ErrorHandler;
 
+use Symfony\Component\ErrorHandler\Exception\SilencedErrorContext;
+
 /**
  * @internal
  */
 class ThrowableUtils
 {
-    public static function getSeverity(\Throwable $throwable): int
+    /**
+     * @param SilencedErrorContext|\Throwable
+     */
+    public static function getSeverity($throwable): int
     {
-        if ($throwable instanceof \ErrorException) {
```

Comment:
Hello. 
Such change (extra check for `instanceof SilencedErrorContext`) will lead other tools (PSR3 compatible loggers), which works with \ErrorException and don't know about Symfony to copy-paste exactly the same statement (`|| $throwable instanceof SilencedErrorContext`) in their implementation. 

Don't you think, that make `SilencedErrorContext extends \ErrorException`  will be better solution?

Reply:
There is no contracts with PSR3 loggers. An entry named exception can hold any type.
Also,extending ErrorException would mean storing a stack trace, which is precisely the thing we want to avoid here.

Reply:
After few hours I found one more place in the symfony error-handler codebase where `|| $ instanceof SilencedErrorContext` may be missed
https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/ErrorHandler/Resources/views/logs.html.php#L20 

>extending ErrorException would mean storing a stack trace, which is precisely the thing we want to avoid here.

SilencedErrorContext has "lightTrace"  so, as I understand the question is not in the performance (moreover $silencedErrorCache works  well)

Correct me if I wrong, you don't want to storing trace only for better VarDumper usage https://github.com/symfony/symfony/pull/19568? 

As I can see Symfony components  designed like "independents" from each other. And this ErrorHandler so cool to be used outside of  Symfony Framework (fe by people like me :) ) . I think it will be better to parametrize such tweaks (as backtrace depth) via `__constructor`  than hardcode them for other component needs.
One more example of my words https://github.com/symfony/symfony/commit/76e79444603f57f7391d1f04411a9d9aa815283b

Let me know if the implementation of such Idea (backtrace depth) has a chance to be merged and I try my best to prepare such PR.

Reply:
Good catch, I updated the template file.
SilencedErrorContext is meant to represent silenced PHP errors.
We want it to be as light as possible, that's why they are very simple value objects and should stay as is.